tests/functional/workloads/cnv/test_cnv_cluster_shutdown_recovery.py

```
@@ -187,41 +187,52 @@ def test_vm_abrupt_graceful_shutdown_cluster(
+        logger.info("Waiting for pods to reach running state")
         wait_for_pods_to_be_running(timeout=500)
 
-        # Check cluster health
+        logger.test_step("Verify cluster and CNV health")
         try:
-            logger.info("Making sure ceph health is OK")
+            logger.info("Running Ceph health check")
             Sanity().health_check(tries=50, cluster_check=False)
         except Exception as ex:
-            logger.error("Failed at cluster health check!!")
+            logger.exception(f"Cluster health check failed: {ex}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)